README.md

```diff
@@ -1,13 +1,24 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was to make a survey implementing useStae in Reactjs. 
+The requirements were to use different types of form inputs, displaying a summary at the end. 
+I created a property search website, taking in requirements from the user and displaying results based on it. A sample API result was used from https://www.booli.se/ and modified for use in this project. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I created components based on question type so that each question type is reusable if needed without having hard coded values. 
+The tutorials provided were of great help, but I also used Google extensively and collaborated with team members to solve problems. 
+For the progress bar, I have used React Bootstrap. 
+I also learnt about storing all useStates in an object according to this documentation https://beta.reactjs.org/reference/react/useState#updating-objects-and-arrays-in-state
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-survey-joannaphilips.netlify.app/
+
+## Image credits
+
+"https://www.flaticon.com/free-icons/apartment" Apartment icons created by amonrat rungreangfangsai - Flaticon
+"https://www.flaticon.com/free-icons/villa" Villa icons created by Freepik - Flaticon
+"https://www.flaticon.com/free-icons/townhouse" Townhouse icons created by Freepik - Flaticon
+Photo by  Damir: https://www.pexels.com/photo/barn-house-beside-tall-trees-during-winter-15483064/
```

Comment:
good readme! Good to keep all these link for future reference and of course good to do the image credits